.eslintrc.js

```diff
@@ -2,6 +2,7 @@ const clientESLintConfig = require('./config/eslint');
 
 module.exports = Object.assign({}, clientESLintConfig, {
   env: Object.assign({}, clientESLintConfig.env, {
-    node: true
+    node: true,
+    jest: true,
```

Comment:
Does that line mean that eslint will allow to write global `describe` even in source folder?